types/src/block_time.rs

```diff
@@ -17,6 +17,7 @@ impl BlockTime {
 
     /// Saturating integer subtraction. Computes `self - other`, saturating at `0` instead of
     /// overflowing.
+    #[must_use]
```

Comment:
Did the compiler suggest you add these `#[must_use]`s?

Reply:
```
error: missing `#[must_use]` attribute on a method returning `Self`
  --> types/src/block_time.rs:21:5
   |
21 | /     pub fn saturating_sub(self, other: BlockTime) -> Self {
22 | |         BlockTime(self.0.saturating_sub(other.0))
23 | |     }
   | |_____^
```